### DISCUSSION TITLE
Multiple configuration files

### DISCUSSION THREAD

**Discussion by Blu3train** [buliasz/tesstrain-windows-gui#31] [Ideas]
I am experimenting with training in English and Japanese text.
To quickly switch between one configuration and another, I saved the "tesstrain_gui.ini" files and swapped them according to the language being trained.

It would be an interesting way to start with the desired configuration file. Maybe an idea could be via command line parameter.
What do you think?

**Comment by buliasz** [comment]
It's a good idea to have some functionality like this. I'd like to avoid command line parameters as a main or the only way to access some functionalities. The idea of this GUI is to avoid requirement of learning and executing command line. 
Anyway I think if we enable such functionality in the GUI, like for example "Load settings" button, we can also add a command line parameter afterwards for those who would like to automate it.
